packages/core/src/middleware/koa-security-headers.ts

```diff
@@ -26,15 +39,82 @@ export default function koaSecurityHeaders<StateT, ContextT, ResponseBodyT>(): M
    * - originAgentCluster: https://whatpr.org/html/6214/origin.html#origin-keyed-agent-clusters
    */
 
-  return helmet({
-    contentSecurityPolicy: false, // Will be set exclusively by the app
+  const basicSecurityHeaderSettings: HelmetOptions = {
+    contentSecurityPolicy: false, // Exclusively set per app
     expectCt: false, // Not recommended, will be deprecated by modern browsers
     dnsPrefetchControl: false,
-    frameguard: false, // TODO  Will be set exclusively by the app
-    crossOriginResourcePolicy: false, // TODO  Will be set exclusively by the app
-    crossOriginEmbedderPolicy: false, // TODO Will be set exclusively by the app
     referrerPolicy: {
       policy: 'strict-origin-when-cross-origin',
     },
-  });
+  };
+
+  const mainFlowUiSecurityHeaderSettings: HelmetOptions = {
+    ...basicSecurityHeaderSettings,
+    // WARNING: high risk Need to allow self hosted terms of use page loaded in an iframe
+    frameguard: false,
+    // Alow loaded by ac preview iframe
```

Comment:
```suggestion
    // Alow loaded by console preview iframe
```
shall we use "console" instead of "ac" for alignment?

Reply:
i can add a lint rule later